src/components/genesys-web-message/GenesysWebMessage.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="d-none"></div>
+</template>
+
+<script lang="ts">
+// Libraries
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+/**
+ * This component creates the button to generate a chat with Service BC Helpdesk.
+ * This component cannot be unit tested because it heavily relies on a third-party library.
```

Comment:
I think this second comment should be in a README.md file in this folder, along with info on how to use this component (ie, a code snippet from App.vue or something). And also some info on the props to provide.